indico/modules/events/editing/controllers/backend/management.py

```diff
@@ -223,3 +225,12 @@ def _process_DELETE(self):
                        'Closed {} editing'.format(orig_string(self.editable_type.title)), session.user)
         editable_type_settings[self.editable_type].set(self.event, 'editing_enabled', False)
         return '', 204
+
+
+class RHContactEditingTeam(RHEditableTypeManagementBase):
+    """Send emails to editing team"""
```

Comment:
```suggestion
    """Send emails to editing team."""
```
To keep it consistent with the rest of the file